CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.h

```diff
@@ -34,7 +34,7 @@ class CaloTPGTranscoderULUT : public CaloTPGTranscoder {
   double hcaletValue(const HcalTrigTowerDetId& hid, const HcalTriggerPrimitiveSample& hc) const override;
   virtual bool HTvalid(const int ieta, const int iphi, const int version) const;
   virtual const std::vector<unsigned int> getCompressionLUT(const HcalTrigTowerDetId& id) const;
-  virtual void setup(HcalLutMetadata const&, HcalTrigTowerGeometry const&, int, int);
+  virtual void setup(HcalLutMetadata const&, HcalTrigTowerGeometry const&, int, int, double, double, bool);
```

Comment:
for basic types like `int`, `double` and `bool` it is usually helpful to provide a parameter name, while this is, of course, not needed for expressive type names like `HcalLutMetadata`

Reply:
Sure!